code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/energy/kinetic_accelerator
 	name = "proto-kinetic accelerator"
-	desc = "A self recharging, ranged mining tool that does increased damage in low pressure. Capable of holding up to six slots worth of mod kits."
+	desc = "A self recharging, ranged mining tool that does increased damage in low pressure. Capable of holding up to three slots worth of mod kits."
```

Comment:
depending on the modkit, it can hold more than three, IE; four range upgrades. Then also a tracer color change, minebot pass through, and super ka chassis. so technically 7, but as those dont take up space, it'd be arguable to count them.

Reply:
should I change something about it then? Because the old description of six was definitely wrong but seems this one is too.

Reply:
honestly, probably better to just mention its upgradeable, since more upgrades can be added that might have a lower cost